Library/Formula/ghi.rb

```diff
@@ -7,6 +7,8 @@ class Ghi < Formula
 
   bottle :unneeded
 
+  depends_on "pygments.rb" => :ruby
```

Comment:
Small heads-up, if a user has a non-homebrew-managed ruby, this will make the formula impossible to install. See https://github.com/Homebrew/legacy-homebrew/issues/25388.